pinot-servicemanager/Dockerfile

```diff
@@ -27,15 +29,15 @@ LABEL MAINTAINER Hypertrace "https://www.hypertrace.org/"
 # Add HEALTHCHECK and ENTRYPOINT scripts into the default search path
 COPY docker-bin/* /usr/local/bin/
 
-# All content including binaries and logs write under WORKDIR
+# Copy the fully configured installation with schema here
 ARG USER=pinot
 WORKDIR /opt/pinot
 
 # Ensure the process doesn't run as root
 RUN adduser -g '' -h ${PWD} -D ${USER}
 USER ${USER}
 
-# Copy binaries and config we installed earlier
+# Copy the fully configured installation with schema here
```

Comment:
duplicate comment?

Reply:
indeed it is fuzz from workspace mess where I was trying to do this as a layer and thwarted by ZK dep https://github.com/apache/incubator-pinot/issues/5977